docs-ref-conceptual/install-azure-cli.md

```diff
@@ -225,7 +225,7 @@ If you used the script at https://aka.ms/InstallAzureCli to install the CLI, you
 2. Delete the line `<install location>/lib/azure-cli/az.completion` from `<install location>/.bash_profile`.
 
 > [!Note]
-> The default install location is `/Users/<username>`.
+> The default install location is /Users/<username> in macOS and /home/<username> in Linux.
```

Comment:
Replace `in` with `for`, and use backticks for formatting of paths.

Reply:
Thanks a lot - I reflected your kind comment!